### PR TITLE
Add feature to allow user to choose which subreddit to pull from

### PR DESCRIPTION
**Pull request by owengaspard**
## What is this?
I have added a feature which can either take user input each run or take the variable in the `.env` file to choose which subreddit to pull from. Both code samples are there, and it is up to the user to choose which option they want.

techsupportgore
This example uses a post from the subreddit techsupportgore.

### CODE REVIEW

**Comment by lvunen** [comment]
Hey Owen, loved this piece of code you added! Could you explain how I can set up the .env file?

**Review by SpaYco — CHANGES_REQUESTED** [review]
Hey, good work, but I see there are some pycache files around, I don't think they should be here.
also make sure to remove `askreddit.py` since we will use `subreddit.py`

**Comment by owengaspard** [comment]
> Hey Owen, loved this piece of code you added! Could you explain how I can set up the .env file?

I added two values to the .env file: "ASK_EACH_TIME" and "SUBREDDIT." Ask Each Time needs to be set to "TRUE" (in quotes) if you want to define the subreddit to pull from each run. "SUBREDDIT" will be ignored if AET is set to "TRUE." If it's set to anything besides that, it will fall back to using AskReddit. If nothing in "SUBREDDIT" is defined while AET is not true, it will also use AskReddit.

TL;DR: you just need to add two lines:
`ASK_EACH_TIME=""`
`SUBREDDIT=""`

**Comment by GKILLA1425** [comment]
> > Hey Owen, loved this piece of code you added! Could you explain how I can set up the .env file?
> 
> I added two values to the .env file: "ASK_EACH_TIME" and "SUBREDDIT." Ask Each Time needs to be set to "TRUE" (in quotes) if you want to define the subreddit to pull from each run. "SUBREDDIT" will be ignored if AET is set to "TRUE." If it's set to anything besides that, it will fall back to using AskReddit. If nothing in "SUBREDDIT" is defined while AET is not true, it will also use AskReddit.
> 
> TL;DR: you just need to add two lines: `ASK_EACH_TIME=""` `SUBREDDIT=""`

Hey owen, do we put the subreddit url in the SUBREDDIT env field?

**Comment by owengaspard** [comment]
> > Sounds great! I would love to see that.
> 
> working on it rn, is there a way to choose a certain voice ? im struggling to find where it gets or chooses a voice from

That's something I looked into before. The library `gTTS` just uses Google Translate for the voices. As far as I know, the only way to change the voice is to change the language.

**Comment by GKILLA1425** [comment]
> > > Sounds great! I would love to see that.
> > 
> > 
> > working on it rn, is there a way to choose a certain voice ? im struggling to find where it gets or chooses a voice from
> 
> That's something I looked into before. The library `gTTS` just uses Google Translate for the voices. As far as I know, the only way to change the voice is to change the language.

yeah i see that now, alright  ill see if i can implement a different tts service

**Comment by elebumm** [comment]
@owengaspard I think this feature is amazing. I did edit the code a little bit, because I think if the user has submitted a subreddit, it shouldn't bother asking them to change it. So I removed that and cleaned up the code.

Thanks for the amazing contribution. It looks fantastic.
